bower.json

```diff
@@ -56,21 +56,24 @@
   },
   "appPath": "app",
   "resolutions": {
-    "angular-bootstrap": "0.14.3",
+    "angular-bootstrap": "2.2.x",
     "d3": "3.5.17",
-    "patternfly": "3.24.0",
-    "angular-patternfly": "3.24.0",
+    "patternfly": "3.25.1",
+    "angular-patternfly": "4.1.1",
     "patternfly-bootstrap-treeview": "2.1.1",
     "angular": "1.5.11",
     "angular-sanitize": "1.5.11",
     "moment": "2.14.2",
     "font-awesome": "4.7.0",
-    "jquery": "2.1.4"
+    "jquery": "3.2.1",
+    "lodash": "4.x",
```

Comment:
I have the same question here: Is the resolution necessary? Is something else depending on Lodash 3.x?